rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -12,20 +12,223 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+find_package(rosidl_generator_c REQUIRED)
+find_package(rmw_implementation_cmake REQUIRED)
+find_package(rmw REQUIRED)
+
+# NOTE(esteve): required for CMake-2.8 in Ubuntu 14.04
+set(Python_ADDITIONAL_VERSIONS 3.4)
+find_package(PythonInterp 3.4 REQUIRED)
+
+find_program(PYTHON_CONFIG_EXECUTABLE NAMES "python3-config")
+if(NOT DEFINED PYTHON_CONFIG_EXECUTABLE)
```

Comment:
This check is wrong since the variable is always defined even when being set to `<VAR>-NOTFOUND`.